Jenkins/InedoExtension/JenkinsArtifactImporter.cs

```diff
@@ -16,6 +15,7 @@ internal sealed class JenkinsArtifactImporter
         public string ArtifactName { get; set; }
         public string JobName { get; set; }
         public string BuildNumber { get; set; }
+        public string SubFolder { get; set; }

```

Comment:
Added this property to contain an optional Jenkins subfolder like 'output/production/appname' to download : -
`https://{jenkins.server.com}/job/{folder-name}/job/{repository-name}/job/{branch}/{build-number}/artifact/{subfolder/child-folder/archive-folder}/*zip*/{archive-folder}.zip`

Examples: -

`https://jenkins.server.com/job/my-organization/job/some-app/job/master/123/artifact/output/production/myapp/*zip*/myapp.zip`